package.json

```diff
@@ -18,15 +18,15 @@
     "linter:fix": "prettier --write '{,!(node_modules|.git|storybook-static)/**/}*.{js,json,jsx,md}'",
     "linter:check": "eslint src/**/* && eslint stories/*.jsx",
     "serve": "start-storybook -p 3006 -s ./public",
-    "build:storybook": "storybook-to-ghpages",
-    "build:static": "build-storybook -s public",
-    "build:package": "babel --delete-dir-on-start --extensions .ts,.tsx,.js,.jsx ./src --out-dir ./dist && tsc --project tsconfig.types.json --emitDeclarationOnly ",
+    "storybook:check": "tsc --project tsconfig.storybook.json --noEmit",
+    "storybook:publish": "npm run storybook:check && storybook-to-ghpages",
```

Comment:
tbh it was like this before I just renamed it to what I think a better name (correct me if I'm wrong here) and just added the extra check step
but yeah it makes sense to also build I guess, should I add it?

Reply:
Aight lets leave it, we can rework it later because it would also change the flow of the CI

Reply:
I think that was a bit of a miss comment from me, I think the command should work independently, and they have dependencies but we shouldnt add all the dependencies in every command. For example here I think we should remove check from publish, because publish expects that you've alreayd run build and test as we do on the CI. If we add check there, we should also add build and test, which is obviously not maintainable